Services/Interfaces/web_interface/flask_util/context_processor.py

```diff
@@ -111,9 +111,16 @@ def get_plugin_tabs(location):
             has_open_source_package = models.has_open_source_package()
             for plugin in web_interface_instance.registered_plugins:
                 for tab in plugin.get_tabs():
-                    if tab.location is location and (not tab.requires_open_source_package or has_open_source_package):
+                    if tab.location is location and tab.is_available(has_open_source_package):
                         yield tab
 
+        def has_strategy_designer_from_cloud():
+            for plugin in web_interface_instance.registered_plugins:
+                for tab in plugin.get_tabs():
+                    if tab.display_name == "Strategy design":
```

Comment:
Don't we have another way to fix it? I'm not sure it's a good idea to hardcode tabs like this